app/views/projects/blob/new.html.haml

```diff
@@ -1,7 +1,13 @@
-- page_title "New File", @ref
-%h3.page-title New file
+%h3.page-title<
+  Create new file or 
+  %a.button{:href=>'#modal-upload-blob', 'data-target'=>'#modal-upload-blob', 'data-toggle'=>'modal'}> &nbspupload existing one
+  
+.file-title
+  =render 'projects/blob/upload'
+  </br>
```

Comment:
@randx , here the tag is used for get the upload modal, I tried other ways, however, this is the one succeed. Which way do you think should  be here?